ui/DesignSystem/Component/SourceBrowser/CommitTeaser.svelte

```diff
@@ -29,15 +29,15 @@
 
   .align-left {
     display: flex;
-    flex: 1;
     align-items: center;
-    max-width: 64%;
+    margin-right: 1ex;
```

Comment:
Why is this `1ex`?

Reply:
It’s there to have a one character space between the message and the author. Could also be `1em` or `2em`.

Reply:
Probably `1rem` would make sense, we use `rem`s everywhere.